### PR TITLE
Update flix-tech/avro-php

### PR DESCRIPTION
**Pull request by MaurizioMoreo**
Update needed to support PHP8.2

### CODE REVIEW

**Comment by tPl0ch** [comment]
@MaurizioMoreo This is where the CI magic is happening:
https://github.com/flix-tech/schema-registry-php-client/blob/master/.github/workflows/checks.yml#L32

File: composer.json [hunk 1]
```diff
@@ -11,14 +11,14 @@
     }
   ],
   "require": {
-    "php": "^7.4|^8.0|^8.1",
+    "php": "^7.4|^8.0|^8.1|^8.2",
```
Comment (marko-ilic) [inline composer.json:14]: Shouldn't be: `"php": "^7.4|^8.0"`? (that covers all 8.x.x versions)

Comment (MaurizioMoreo) [reply composer.json:14]: Yes, the caret should cover until the next major version. I wanted to be conservative but let's remove it :+1:

**Comment by MaurizioMoreo** [comment]
Are the expected `ci-checks (8, --prefer-stable)` and `ci-checks (8, --prefer-lowest)` related to testing the behaviour when we don't specify the minor version? If yes, do you think we should add it?

**Comment by MaurizioMoreo** [comment]
Any news on this PR @marko-ilic @tPl0ch ?

**Review by marko-ilic — APPROVED** [review]

**Comment by marko-ilic** [comment]
> Any news on this PR @marko-ilic @tPl0ch ?

It looks good to me, but I'm not sure who else should check/approve it

**Comment by MaurizioMoreo** [comment]
It seems that only @tPl0ch can merge it, but he is a magical creature that maybe only @nick-zh can awaken :magic_wand: :sweat_smile:

**Comment by tPl0ch** [comment]
Hey all, I'll have time tomorrow to check it. Sorry for the delay.

**Comment by tPl0ch** [comment]
Released in https://github.com/flix-tech/schema-registry-php-client/releases/tag/8.0.0

**Pull request merged**